### DISCUSSION TITLE
Welcome to FuzzyMail Discussions!

### DISCUSSION THREAD

**Discussion by luangjokaj** [luangjokaj/fuzzymail#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zalexstudios** [comment]
https://prnt.sc/12ol31z
Where is it spelled out?

**Reply by luangjokaj** [reply]
They are 2 different colors. https://cherry.design/docs/code/postcss/colors

**Reply by zalexstudios** [reply]
Thanks for the answer!
Could you make a YouTube video for the original setting FuzzyMail? Or make some light project on your assembly on gulp.
Thanks for your work!
